packages/esbuild/esbuild.bzl

```diff
@@ -15,6 +15,9 @@ def _esbuild_impl(ctx):
     # how to resolve custom package or module names
     path_alias_mappings = dict()
 
+    if (ctx.attr.link_workspace_root):
+        path_alias_mappings.update(generate_path_mapping(ctx.workspace_name, "/".join(ctx.build_file_path.split("/")[:-1])))
```

Comment:
This would add:

```
"build_bazel_rules_nodejs": [
    "packages/esbuild/test/typescript/index.mjs",
    "packages/esbuild/test/typescript"
],
"build_bazel_rules_nodejs/*": [
    "packages/esbuild/test/typescript/*"
]
```

to the path mappings. Wouldn't these need to point to the root of the workspace, which would be `'./*'` rather than the root of where the `esbuild` file is?

Reply:
Yeah you're right... the import examples in the test should be `build_bazel_rules_nodejs/packages/esbuild/test/typescript/...subfolder...`. I think that's actually what I originally had but was getting errors and confused (all because of a silly typo), and this is what came out of the confusion in the end...

Reply:
Updated, I think that's right now?

Reply:
For the test this now creates:
```
"paths": {
            "build_bazel_rules_nodejs": [
                "./index.mjs",
                "."
            ],
            "build_bazel_rules_nodejs/*": [
                "./*"
            ]
        }
```

Reply:
Yeah that looks correct, nice one!